google/cloud/security/common/util/email_util.py

```diff
@@ -20,10 +20,11 @@
 
 import jinja2
 
-from retrying import retry
 import sendgrid
 from sendgrid.helpers import mail
 
+from retrying import retry
```

Comment:
retrying is an installed 3rd party library just like jinja and sendgrid; perhaps try grouping all of them together?